src/Umbraco.Core/Services/Implement/ContentService.cs

```diff
@@ -1387,11 +1453,14 @@ private IEnumerable<PublishResult> PerformScheduledPublishInternal(DateTime date
                             .ToList();
 
                         if (pendingCultures.Count == 0)
-                            break; //shouldn't happen but no point in continuing if there's nothing there
+                            continue; //shouldn't happen but no point in continuing if there's nothing there
```

Comment:
maybe update/remove these comments .. someone will be confused in the future if it says no point in continuing and we then `continue` 😃 ..